src/test-run/browser-manipulation-queue.js

```diff
@@ -77,13 +77,15 @@ export default class BrowserManipulationQueue {
                     customPath:     command.path,
                     pageDimensions: driverMsg.pageDimensions,
                     cropDimensions: driverMsg.cropDimensions,
-                    markSeed:       command.markSeed
+                    markSeed:       command.markSeed,
+                    fullPage:       command.fullPage || void 0
```

Comment:
`command.fullPage || void 0`.

It's a strange instruction. Can we avoid it?

Reply:
It's required because we need 3 states of `fullPage` option.
We can think about of refactoring this code. I agree that is's confusing. If we decide to refactor this, we will need to add nullable boolean argument to the command

Reply:
It's no more required. Hooray